db/models.py

```diff
@@ -14,3 +14,43 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movie_actors")
+    genres = models.ManyToManyField(Genre, related_name="movie_genres")
```

Comment:
The `related_name` is not correct for this field, it should not be a case to get all movies from `Actor` by referring to `movie_actors`, and the same for `Genre`. It would be best if you got all movies from `Actor` or `Genre` by using `movies` as a `related_name`, make this change here.